recOrder/cli/apply_inverse_transfer_function.py

```diff
@@ -26,348 +33,377 @@ def _check_background_consistency(background_shape, data_shape):
         )
 
 
-def apply_inverse_transfer_function_cli(
-    input_data_path, transfer_function_path, config_path, output_path
+def apply_inverse_biref_phase_2D(
+    czyx, birefringence_2D_args, birefringence_3D_args, phase2D_args
 ):
-    echo_headline("Starting reconstruction...")
+    reconstruction_birefringence_2D = (
+        inplane_oriented_thick_pol3d.apply_inverse_transfer_function(
+            czyx, **birefringence_2D_args
+        )
+    )
+    reconstruction_birefringence_3D = (
+        inplane_oriented_thick_pol3d.apply_inverse_transfer_function(
+            czyx, **birefringence_3D_args
+        )
+    )
+    brightfield_3d = reconstruction_birefringence_3D[2]
+    (
+        _,
+        yx_phase,
+    ) = isotropic_thin_3d.apply_inverse_transfer_function(
+        brightfield_3d, **phase2D_args
+    )
+    zyx_phase = yx_phase.unsqueeze(0)
 
-    # Load datasets
-    transfer_function_dataset = open_ome_zarr(transfer_function_path)
-    input_dataset = open_ome_zarr(input_data_path)
+    reconstruction_bire_phase_2D = reconstruction_birefringence_2D + (
+        zyx_phase,
+    )
+    return reconstruction_bire_phase_2D
 
-    # Load config file
-    settings = utils.yaml_to_model(config_path, ReconstructionSettings)
 
-    # Check input channel names
-    if not set(settings.input_channel_names).issubset(
-        input_dataset.channel_names
-    ):
-        raise ValueError(
-            f"Each of the input_channel_names = {settings.input_channel_names} in {config_path} must appear in the dataset {input_data_path} which currently contains channel_names = {input_dataset.channel_names}."
+def apply_inverse_biref_phase_3D(czyx, birefringence_args, phase3D_args):
```

Comment:
#396. Functions like this. This still needs some standardization as it still returns a tuple of tensors rather than an array.